### ISSUE TITLE
Class order in the generated stubs file doesn't allow them to be loaded properly

### ISSUE THREAD

**Issue by zaantar** [GiacoCorsiglia/php-stubs-generator#1]
Hi there, first of all, let me say that this is an awesome tool that has saved us a lot of time. So, thanks. :)

However, we've run into some issues when trying to use [generated WordPress stubs](https://github.com/GiacoCorsiglia/wordpress-stubs) together with [PHPStan](https://github.com/phpstan/phpstan) to analyze a WordPress plugin we're developing.

PHPStan, probably unlike other static analysis tools out there, is a bit less "static" and needs to actually load the php files in order to work.

That is not possible because the generated stub file contains things like this:

```php
class Subclass extends ParentClass {

}

class ParentClass {

}
```

We need the parent class to be defined first, otherwise a PHP error occurrs when loading the file.

**Comment by GiacoCorsiglia** [comment]
Hello!  Glad to hear you've found a use for this.  Can't believe I never tried to execute the generated stubs; I guess they worked for Psalm which was enough for me.  I'll try to implement a fix this weekend.

Of interest, your example code actually works fine (though it demonstrated your point).  You need 2 levels of inheritance, both declared in the wrong order, for PHP to complain:

```php
class SubSubClass extends SubClass {}
class SubClass extends ParentClass {}
class ParentClass {}
```

Coincidentally, there was just a [Reddit post](https://www.reddit.com/r/PHP/comments/84c4nm/eli5_why_when_extending_a_class_the_order_of/) related to this.

Anyway, thanks for reporting this issue!

**Comment by GiacoCorsiglia** [comment]
Hi @zaantar, I've released an [updated version](https://github.com/GiacoCorsiglia/wordpress-stubs/releases/tag/v4.9.4.2) of the WordPress stubs which fixes this issue along with several others.  You can now run `php wordpress-stubs.php` without any fatal errors; however, there are still quite a few notices or warnings due to the global variable declarations.

Could you let me know if this is enough for PHPStan?  If not, one solution may be to initialize all stubbed global variables as `null`, though I could see static analysis tools complaining about this as well.

**Comment by zaantar** [comment]
Thank you a lot, @GiacoCorsiglia! I will play with PHPStan a bit more and let you know about my findings.
